<launch>


      <param name="/use_sim_time" value="true" />

<node name="turtlebot_controller" pkg="turtlebot_trajectory_controller" type="trajectory_controller_node" output="screen" args="" >
    <remap from="odom" to="visual/odom"/>
</node>


<!--
<include file="$(find nav_scripts)/launch/egocylindrical_pips_dwa_controller.launch">
  <arg name="odom_topic" value="visual/odom"/>
</include>
-->

  <node name="turtlebot_trajectory_testing" pkg="turtlebot_trajectory_testing" type="send_test_trajectory" output="screen" args="_fw_vel:=0.7 _waypoint_yaml:=/home/yipuzhao/catkin_ws/src/turtlebot_trajectory_testing/config/square.yaml" >
  </node>

</launch>